types/components/Collapse.d.ts

```diff
@@ -4,6 +4,8 @@ import { TransitionCallbacks } from './helpers';
 export interface CollapseProps
   extends TransitionCallbacks,
     React.ClassAttributes<Collapse> {
+  id?: string;
+  className?: string
```

Comment:
they don't belong on this type Collapse, they belong on NavbarCollapse, and then they shouldn't be listed individually they should extend the react div props

Reply:
see #2785

Reply:
Just FYI,

@julionadal A huge discussion was done on [this thread](https://github.com/react-bootstrap/react-bootstrap/pull/3448/commits/57278de982fb03e9b2e55a305f6f28379810706f#r256593836) in regards to the same issue. The decision to rebuild the API was made by @jquense and @thejohnfreeman

Reply:
Also this is a duplicate of #3502

Reply:
Thank you, i just saw @prateekkathal. Hope this changes are going to be released soon.